lib/ansible/plugins/strategy/__init__.py

```diff
@@ -75,7 +75,7 @@ def results_thread_main(strategy):
     while True:
         try:
             result = strategy._final_q.get()
-            if type(result) == object:
+            if type(result) is type(_sentinel):
```

Comment:
Not sure this is correct (before or after patch), since the sentinel check is usually checking object identity.
I would expect this to be:

``` python
_sentinel = object()
 def results_thread_main(strategy):
      while True:
          try:
              result = strategy._final_q.get()
              if result is _sentinel:
                  break
```